# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fget/hotfix-bin-sh.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fget/warn-auto.sh.vanilla	2023-12-05 23:14:55.718850531 +0100
+++ fget/warn-auto.sh	2023-12-05 23:15:00.276850336 +0100
@@ -1,2 +1,2 @@
-#!/bin/sh
+#!/bin/bash
 # WARNING: This file was auto-generated. Do not edit!
